diff --git a/node_modules/@react-email/render/dist/browser/index.mjs b/node_modules/@react-email/render/dist/browser/index.mjs
index 0d63ab6..74e9b0d 100644
--- a/node_modules/@react-email/render/dist/browser/index.mjs
+++ b/node_modules/@react-email/render/dist/browser/index.mjs
@@ -155,9 +155,9 @@ var render = (element, options) => __async(void 0, null, function* () {
   const suspendedElement = /* @__PURE__ */ jsx(Suspense, { children: element });
   const reactDOMServer = yield import("react-dom/server");
   let html2;
-  if (Object.hasOwn(reactDOMServer, "renderToReadableStream")) {
+  if (Object.hasOwn(reactDOMServer.default, "renderToReadableStream")) {
     html2 = yield readStream(
-      yield reactDOMServer.renderToReadableStream(suspendedElement)
+      yield reactDOMServer.default.renderToReadableStream(suspendedElement)
     );
   } else {
     yield new Promise((resolve, reject) => {
